public/css/style.css

```diff
@@ -1,19 +1,56 @@
 /****************
 *** Variables ***
 ****************/
-html {
-  height: 100%;
-  margin: 0;
-}
-
```

Comment:
inb4 "why is this removed", every page includes normalize.css, which already does this.